src/features/Treege/components/Forms/FormTreeCardMutation/useFormTreeCardMutation.ts

```
@@ -372,20 +404,48 @@ const useFormTreeCardMutation = () => {
     [currentHierarchyPointNode?.data.attributes?.tree, fetchWorkflow, open, t],
   );
 
+  const handleValueFromAncestor = (sourceValue?: string) => {
+    setDefaultValueFromAncestor((prevState) => {
+      const nextState = { ...(prevState ?? {}) };
+
+      if (sourceValue) {
+        nextState.sourceValue = sourceValue;
+      } else {
+        delete nextState.sourceValue;
+      }
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)